libdnf-cli/argument_parser.cpp

```diff
@@ -302,6 +302,67 @@ void ArgumentParser::Command::parse(const char * option, int argc, const char *
     }
 }
 
+// Prints a completed argument `arg` or a table with suggestions and help to complete
+// if there is more than one solution.
+static void print_complete(
+    ArgumentParser::Command * cmd, const char * arg, std::vector<ArgumentParser::NamedArg *> named_args) {
+
+    libdnf::cli::output::Help help;
```

Comment:
Since you're "abusing" the Help class for a different purpose than originally intended, what about adding a short comment along the lines of:
```
// Using the Help class to print the completion suggestions, as it prints a table of two columns which is also what we need here.
```

Reply:
I use the Help class to print help for potential matches for more than one match. But I added a suggested comment.